### PR TITLE
Fix Irix lens names in pentaxmn_int.cpp

### LINKED ISSUE #1084: Irix lens names in Pentax
Hello!
It's not a bug, it's just missing information about Irix150mm, Irix11mm lenses in photos taken with Pentax. I suggest the following patch to add this information:
[Pentax-Irix.patch.zip](https://github.com/Exiv2/exiv2/files/4060622/Pentax-Irix.patch.zip)

### PR DESCRIPTION
**Pull request by D4N**
This fixes #1084.

### CODE REVIEW

**Review by piponazo — APPROVED** [review]
LGTM!

**Comment by D4N** [comment]
I still need to add the reproducers that were reported in #1084.

**Comment by clanmills** [comment]
Let's see if we can include this in v1.00 and add test code.

File: src/pentaxmn_int.cpp [hunk 1]
```diff
@@ -776,7 +776,9 @@ namespace Exiv2 {
         { 0x0402, "smc PENTAX-FA 80-320mm F4.5-5.6" },
         { 0x0403, "smc PENTAX-FA 43mm F1.9 Limited" },
         { 0x0406, "smc PENTAX-FA 35-80mm F4-5.6" },
-        { 0x040a, "Irix 15mm F2.4" },
+	{ 0x0408, "Irix 150mm F/2.8 Macro" },
+	{ 0x0409, "Irix 11mm F/4" },
```
Comment (neheb) [inline src/pentaxmn_int.cpp:780]: whitespace

**Comment by kmilos** [comment]
Superseded by https://github.com/Exiv2/exiv2/pull/1118

**Pull request closed without merging**